### ISSUE TITLE
Computer name not changing when hostname has been updated

### ISSUE THREAD

**Issue by pLeXwithFleX** [chocolatey/chocolatey-licensed-issues#368]
### Checklist

- [X] I have verified this is the correct repository for opening this issue.
- [X] I have verified no other issues exist related to my request.

### Is Your Feature Request Related To A Problem? Please describe.

Currently the computer object name on central management is not changing when the hostname was changed. The computer is not clearly to identified.

### Describe The Solution. Why is it needed?

When the chocolatey agent is polling it could send and compare / update the computer object name.

┆Issue is synchronized with this [Gitlab issue](https://gitlab.com/chocolatey/central-management/choco-licensed-management-ui/-/issues/942) by [Unito](https://www.unito.io)

**Comment by pauby** [comment]
So we can capture all the information for this, can I ask you to raise a new issue in this repository, using the correct template (which will ask you for all the information needed) and referencing this issue? Once that is done, we can close this issue (as it will no longer be needed).

**Comment by pLeXwithFleX** [comment]
Hi pauby,

youre welcome, Im sorry for posting to the wrong repository. I will create a new issue.
